pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp

```diff
@@ -252,8 +252,12 @@ void Dialogs::CreateMemoryCardDialog::CreateControls()
 		.	SetToolTip(_t("Use at your own risk.  Erratic memory card behavior is possible (though unlikely)."))
 		.	SetInt(64),
 
-		RadioPanelItem(_("Folder [experimental]"), _("Store memory card contents in the host filesystem instead of a file."))
-		.	SetToolTip(_t("Automatically manages memory card contents so that the console only sees files related to the currently running software. Allows you to drag-and-drop files in and out of the memory card with your standard file explorer. This is still experimental, so use at your own risk!"))
+		RadioPanelItem(_("Folder [best with PCSX2]"), _("Transforms the memory card into an usable folder with files inside."))
```

Comment:
Not sure I like "best with PCSX2", I think "Recommended" would fit the same theme as the rest of the application. Usually we just say "Recommended" or "Not Recommended". 

"Transforms the memory card into an usable folder with files inside." - I do not like this, it does not really say anything relevant to what the folder memory card is. I much prefer the original to this.